src/parser/monk/mistweaver/modules/talents/ManaTea.js

```diff
@@ -162,17 +169,45 @@ class ManaTea extends Analyzer {
     };
   }
 
+  get avgOverHealing(){
+    let overHealingPercent = this.overhealing/(this.overhealing+this.effectivehealing);
+    overHealingPercent = Math.round(overHealingPercent * 1000)/1000;
+    return overHealingPercent;
+  }
+
+  get suggestionThresholdsMana(){
+    return {
+      actual: this.avgOverHealing,
+      isGreaterThan: {
+        minor: .20,
+        average: .30,
+        major: .40,
+      },
+      style: 'percentage',
+    };
+  }
+
   suggestions(when) {
     when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) => {
       return suggest(
         <>
-          Your mana spent during <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> can be improved. Always aim to cast your highest mana spells such as <SpellLink id={SPELLS.ESSENCE_FONT.id} /> or <SpellLink id={SPELLS.VIVIFY.id} />.
+          Your mana spent during <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> can be improved. Aim to only cast <SpellLink id={SPELLS.VIVIFY.id} /> untill the last second then cast <SpellLink id={SPELLS.ESSENCE_FONT.id} />.
         </>
       )
         .icon(SPELLS.MANA_TEA_TALENT.icon)
         .actual(`${formatNumber(this.avgMtSaves)} average mana saved per Mana Tea cast`)
         .recommended(`${(recommended / 1000).toFixed(0)}k average mana saved is recommended`);
     });
+    when(this.suggestionThresholdsMana).addSuggestion((suggest, actual, recommended) => {
+      return suggest(
+        <>
+          Your overhealing was high during your <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> usages. Consider using <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> during periods of high damage to the raid and also look to target low health raid members.
```

Comment:
Can this be slightly reworded:
`Your average overhealing was high during your <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> usage. Consider using <SpellLink id={SPELLS.MANA_TEA_TALENT.id} /> during specific boss abilities or general periods of high damage to the raid. Also look to target low health raid members to avoid large amounts of overhealing,`